README.md

```diff
@@ -227,60 +231,64 @@ Please note, that
 - `queryTest` is also called if you use `@XrayTest` annotation, but without key attribute
 - `queryTestSet` is also called if you `@XrayTestSet` annotation, but without key attribute
 
-#### Update entities
+#### Creating new entities
+
+By default, the Xray connector doesn't create any issues. You can enable that by passing `true` in the interface.
 
-The `XrayMapper` also provides callbacks for updating entities.
+Please note, that existing issues will be updated automatically. All manual changes like test steps will be overwritten.
 
 ```java
 public class GenericMapper implements XrayMapper {
     
     @Override
-    public void updateTestExecution(XrayTestExecutionIssue xrayTestExecutionIssue, ExecutionContext executionContext) {
-        xrayTestExecutionIssue.getTestEnvironments().add("Test");
-        xrayTestExecutionIssue.setFixVersions(List.of(new JiraNameReference("1.0")));
+    public boolean shouldCreateNewTestSet(ClassContext classContext) {
+        return true;
     }
 
     @Override
-    public void updateTestSet(XrayTestSetIssue xrayTestSetIssue, ClassContext classContext) {
-        xrayTestSetIssue.getLabels().add("TestAutomation");
+    public boolean shouldCreateNewTest(MethodContext methodContext) {
+        return true;
     }
 
     @Override
-    public void updateTest(XrayTestIssue xrayTestIssue, MethodContext methodContext) {
-        xrayTestIssue.getLabels().add("TestAutomation");
+    public String getDefaultTestIssueSummery(MethodContext methodContext) {
+        return String.format("%s_%s", methodContext.getClassContext().getName(), methodContext.getName());
     }
 }
 ```
 
-You can use these methods to update the Jira issues right before importing. Please mind, that not all features are supported by the [Xray import API](#references).
+If you create new test issues, Xray connector will use the method `getDefaultTestIssueSummery` for generate new issue summary.
 
-#### Creating new entities
+In the example above new created test issues get the summery according the format `<TestClass_TestMethod>` like `MyTestClass_testSomething`. 
```

Comment:
"[...] the summ[a]ry according [to ]the format `<TestClass_TestMethod>`[, e.g.] `MyTestClass_testSomething`."